`+response_property.exclusion_rule_groups=[]` no longer works (AssertionError: Exclusion rule group should be list, but is <class 'omegaconf.listconfig.ListConfig'>)

Since #18 I see
```
Error executing job with overrides: ['study_name=jailbreaks', 'task.set=val', 'language_model=gpt-3.5-turbo', 'task=harmbench', 'response_property=jailbroken', 'limit=1', 'strings_path=none', 'language_model.logprobs=5', '+response_property.exclusion_rule_groups=[]', 'prompt=meta_level/really_minimal']
Traceback (most recent call last):
  File "/home/rajashree/introspection_self_prediction_astra/evals/run_meta_level.py", line 262, in main
    asyncio.run(async_main(cfg))
  File "/usr/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/rajashree/introspection_self_prediction_astra/evals/run_meta_level.py", line 212, in async_main
    new_filename = setup_data_file(cfg, exp_dir, filename)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rajashree/introspection_self_prediction_astra/evals/run_meta_level.py", line 245, in setup_data_file
    new_filename = generate_few_shot_data(
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rajashree/introspection_self_prediction_astra/evals/generate_few_shot.py", line 68, in generate_few_shot_data
    object_df = list(load_and_prep_dfs([base_data_path]).values())[0]
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rajashree/introspection_self_prediction_astra/evals/analysis/loading_data.py", line 189, in load_and_prep_dfs
    assert isinstance(
AssertionError: Exclusion rule group should be list, but is <class 'omegaconf.listconfig.ListConfig'>

```

--- a/evals/analysis/loading_data.py
+++ b/evals/analysis/loading_data.py
@@ -186,8 +186,8 @@ def load_and_prep_dfs(
         if isinstance(resp_compliance_group, str):
             resp_compliance_group = eval(resp_compliance_group)
         if resp_compliance_group is not None:
-            assert isinstance(
-                resp_compliance_group, list
+            assert isinstance(resp_compliance_group, list) or isinstance(
+                resp_compliance_group, omegaconf.listconfig.ListConfig
             ), f"Exclusion rule group should be list, but is {type(resp_compliance_group)}"
             for group in resp_compliance_group:
                 compliance_groups.add(group)

